Prog/Calculate population by age and raceethnicity.sas

```diff
@@ -129,11 +140,11 @@ run;
 
 
 proc summary data = Race_&year. ;
```

Comment:
i think you need to put in a proc sort before the proc summary - I'm getting an error that the variables aren't properly sorted.